content/en/graphing/widgets/change.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Change Widget
+kind: documentation
+description: Graph the change in a value over a chosen time period .
+further_reading:
+- link: "graphing/dashboards/timeboard/"
+  tag: "Documentation"
+  text: "Timeboards"
+- link: "graphing/dashboards/screenboard/"
+  tag: "Documentation"
+  text: "Screenboard"
+---
+The Change graph shows you the change in a value over the time period chosen:
+
+{{< img src="graphing/widgets/change/change.png" alt="Change graph" responsive="true">}}
+
+## Setup
+
+{{< img src="graphing/widgets/change/change_setup.png" alt="Change graph Setup" responsive="true" style="width:80%;">}}
+
+### Configuration
+
+1. Choose a metric to graph. 
+2. Choose an aggregation function.
+3. Optional - choose a specific context for your widget.
+4. Break down your aggregation on a Tag Key i.e `host`, `service`..
+5. Choose the Compared period between:
```

Comment:
```suggestion
5. Choose the "Compared" period from:
```

"between" is always between two choices. for three or more choices, use "from" or "among"